boards/riscv/hpm6750evkmini/hpm6750evkmini-pinctrl.dtsi

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2022 hpmicro
```

Comment:
Please make copyright notices consistent across this PR, and format them in a way that is used across Zephyr, e.g. here it should be `Copyright (c) 2022 HPMicro`.